velox/type/TimestampConversion.cpp

```diff
@@ -158,10 +158,11 @@ bool isValidWeekOfMonthDate(
   }
 
   int64_t daysSinceEpochOfFirstDayOfMonth;
-  const Status status =
-      daysSinceEpochFromDate(year, month, 1, daysSinceEpochOfFirstDayOfMonth);
-  if (!status.ok()) {
+  Expected<int64_t> expected = daysSinceEpochFromDate(year, month, 1);
+  if (expected.hasError()) {
     return false;
+  } else {
```

Comment:
Perhaps remove the 'else' after 'return'. Ditto for the others.

Reply:
Thanks. Updated.